### ISSUE TITLE
Paths aren't recognized correctly 

### ISSUE THREAD

**Issue by Efratror** [jonnyboyC/kos-language-server#11]
Currently paths starting with a kos drive number like ```"0:/"``` or from within a subfolder to the root folder, like:
```
*LIB_Foo.ks in "0:/libs"
Runpath("bar.ks")
```
Give a error message complaining visual code can't find the file. Because it sees the drive number as a folder name and it looks for the file in the subfolder with out checking the root folder

**Comment by jonnyboyC** [comment]
Thanks for reporting! Do you have a repo where this occurs I could test against? I'll look into this as soon as possible

**Comment by Efratror** [comment]
sorry for my late reaction. i can't reproduce the error any more. But some other weird things are happening now.  I can't see any error at all. vs code has some UnhandeldPromiseRejectionWarning in it's output while it's parsing

```
[Server(15192) {"workspace":{"applyEdit":true,"workspaceEdit":{"documentChanges":true,"resourceOperations":["create","rename","delete"],"failureHandling":"textOnlyTransactional"},"didChangeConfiguration":{"dynamicRegistration":true},"didChangeWatchedFiles":{"dynamicRegistration":true},"symbol":{"dynamicRegistration":true,"symbolKind":{"valueSet":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26]}},"executeCommand":{"dynamicRegistration":true},"configuration":true,"workspaceFolders":true},"textDocument":{"publishDiagnostics":{"relatedInformation":true},"synchronization":{"dynamicRegistration":true,"willSave":true,"willSaveWaitUntil":true,"didSave":true},"completion":{"dynamicRegistration":true,"contextSupport":true,"completionItem":{"snippetSupport":true,"commitCharactersSupport":true,"documentationFormat":["markdown","plaintext"],"deprecatedSupport":true,"preselectSupport":true},"completionItemKind":{"valueSet":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25]}},"hover":{"dynamicRegistration":true,"contentFormat":["markdown","plaintext"]},"signatureHelp":{"dynamicRegistration":true,"signatureInformation":{"documentationFormat":["markdown","plaintext"]}},"definition":{"dynamicRegistration":true},"references":{"dynamicRegistration":true},"documentHighlight":{"dynamicRegistration":true},"documentSymbol":{"dynamicRegistration":true,"symbolKind":{"valueSet":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26]},"hierarchicalDocumentSymbolSupport":true},"codeAction":{"dynamicRegistration":true,"codeActionLiteralSupport":{"codeActionKind":{"valueSet":["","quickfix","refactor","refactor.extract","refactor.inline","refactor.rewrite","source","source.organizeImports"]}}},"codeLens":{"dynamicRegistration":true},"formatting":{"dynamicRegistration":true},"rangeFormatting":{"dynamicRegistration":true},"onTypeFormatting":{"dynamicRegistration":true},"rename":{"dynamicRegistration":true,"prepareSupport":true},"documentLink":{"dynamicRegistration":true},"typeDefinition":{"dynamicRegistration":true},"implementation":{"dynamicRegistration":true},"colorProvider":{"dynamicRegistration":true},"foldingRange":{"dynamicRegistration":true,"rangeLimit":5000,"lineFoldingOnly":true}}}] Started and initialize received
[Server(15192) c:\Program Files (x86)\Steam\steamapps\common\Kerbal Space Program Modded\Ships\Script] Started and initialize received

(node:15192) UnhandledPromiseRejectionWarning: TypeError: perf_hooks_1.performance.getEntriesByName is not a function or its return value is not iterable
    at Analyzer.<anonymous> (c:\Users\Efratror\.vscode\extensions\johnchabot.kos-vscode-0.2.0\server\out\analyzer.js:214:63)
    at Generator.next (<anonymous>)
    at c:\Users\Efratror\.vscode\extensions\johnchabot.kos-vscode-0.2.0\server\out\analyzer.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (c:\Users\Efratror\.vscode\extensions\johnchabot.kos-vscode-0.2.0\server\out\analyzer.js:3:12)
    at Analyzer.parseDocument (c:\Users\Efratror\.vscode\extensions\johnchabot.kos-vscode-0.2.0\server\out\analyzer.js:188:16)
    at Analyzer.<anonymous> (c:\Users\Efratror\.vscode\extensions\johnchabot.kos-vscode-0.2.0\server\out\analyzer.js:41:82)
    at Generator.next (<anonymous>)
    at c:\Users\Efratror\.vscode\extensions\johnchabot.kos-vscode-0.2.0\server\out\analyzer.js:7:71
    at new Promise (<anonymous>)
(node:15192) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:15192) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Scanning file:///c%3A/Program%20Files%20%28x86%29/Steam/steamapps/common/Kerbal%20Space%20Program%20Modded/Ships/Script/TPid.ks

Parsing file:///c%3A/Program%20Files%20%28x86%29/Steam/steamapps/common/Kerbal%20Space%20Program%20Modded/Ships/Script/TPid.ks
```

**Comment by jonnyboyC** [comment]
Thanks for the comment! I believe this had to do with the performance tracking library functions I've been using. I found out it was removed in node.js 10. My guess is this occurred after updated vscode. I'll have to cut a new release with the updated api. Thanks for bringing this my attention.

After I push an update we can figure out the rest of your issues.

**Comment by Efratror** [comment]
I'm looking forward to your new release.

**Comment by jonnyboyC** [comment]
I Just pushed the 0.3.0. In it I did find and fix an issue related to correctly determining run instruction. Let me know if the 0.3.0 release solves your issue. Otherwise let me know any relevant information so I can track down the root cause

**Comment by Efratror** [comment]
With the release 0.3.0 this issue is fixed see pull #15

**Comment by jonnyboyC** [comment]
Glad to hear!
